docs/UserGuide.adoc

```diff
@@ -250,26 +250,27 @@ __Constraints__
 
 ==== Removing tags from multiple contacts : `detag`
 
-To delete the specified tag of persons identified by the index number, use: +
+When a tag is no longer in used, you can use the detag command to remove the tag from multiple contacts. +
 
-Format: `detag [INDEX]... [t/TAG]`
+Format: `detag INDEX... [t/TAG]`
 
-For example:
+For examples:
 
 * `list` +
 `detag 2 t/friends` +
-Deletes the `friends` tag of the 2nd person in the address book.
+Deletes the [`friends`] tag of the 2nd contact in the address book.
 
 * `find Betsy` +
 `detag 1, 2, 4 t/OwesMoney` +
-Deletes the `OwesMoney` tag of the 1st, 2nd and 4th person in the results of the `find` command.
+Deletes the [`OwesMoney`] tag of the 1st, 2nd and 4th contact in the result list of the `find` command.
 
 __Constraints__
 
-****
-* The index refers to the index number shown in the most recent listing.
-* The index *must be a positive integer* 1, 2, 3, ...
-****
+Note the following constraints when trying to remove tag: +
```

Comment:
remove *tag(s)

Reply:
hmm detag can only remove one at a time right, so would it be better to use just tag?

Reply:
Oh yeah just leave it as tag then